.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "vendor/llhttp"]
     path = vendor/llhttp
     url = https://github.com/nodejs/llhttp.git
-    branch = v8.1.1
+    branch = v8.x
```

Comment:
This shouldn't be a moving target but a more specific thing like a tag or even a commit sha...

Reply:
Again, from previous PR, this is just the branch that it updates from, there is no v8.1.1 branch. The pin is the commit of the actual submodule at vendor/llhttp.

Reply:
Oh, I didn't realize that, somehow. Just checked out https://stackoverflow.com/a/18797720/595220, and I now understand better how this works and why https://github.com/aio-libs/aiohttp/pull/7346 didn't work…